src/audio/src.c

```diff
@@ -61,6 +61,8 @@
 #include <stdio.h>
 #endif
 
+void sys_comp_src_init(void);
```

Comment:
Declarations may be just before definitions. It's anyway just to satisfy compiler, these symbols are not directly used in other places

Reply:
Sure, done